srv/_modules/cluster.py

```diff
@@ -48,18 +45,22 @@ def storage_device_config():
     ]
     for node in server_nodes:
         cluster_dict = provisioner.pillar_get(f"cluster/{node}/roles", targets=node)
+
         if "primary" in cluster_dict[node][f"cluster/{node}/roles"]:
-            cmd = "multipath -ll | grep prio=50 -B2|grep mpath|sort -k2.2 | awk '{ print $1 }'"
+            cmd = ("multipath -ll | grep prio=50 -B2 |"
```

Comment:
I am not sure that it can be passed to `run_subprocess_cmd` since it will split the string command into space-separated sub-parts. I suggest to make split logic optional, e.g. only if shell=False

Reply:
@andkononykhin Checked this - and it is as expected. Since we're not passing cmd as string, it doesn't split and takes as such.
`Command to execute: multipath -ll | grep prio=50 -B2 | grep mpath|sort -k2.2 | awk '{ print $1 }'`